DataConnectors/microsoft-logstash-output-azure-loganalytics/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure Log Analytics output plugin for Logstash 
+
+Azure Sentinel provides a new output plugin for Logstash. Using this output plugin, you will be able to send any log you want using Logstash to the Azure Sentinel/Log Analytics workspace
+Today you will be able to send messages to custom logs table that you will define in the output plugin. 
+Getting started with Logstash 
```

Comment:
add link to https://www.elastic.co/guide/en/logstash/current/getting-started-with-logstash.html